.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
```suggestion
labels: 'feature request'
```
Added this label.

Reply:
Not completely sure if this will work.
The label does not show up in the table when using the "rich diff" view.